common/src/main/java/com/wynntils/gui/screens/maps/GuildMapScreen.java

```diff
@@ -122,6 +142,22 @@ protected void renderPois(
 
         List<Poi> filteredPois = getRenderedPois(pois, textureBoundingBox, poiScale, mouseX, mouseY);
 
+        if (territoryDefenseFilterLevel != TerritoryDefenseLevel.OFF) {
```

Comment:
You should do this in `renderPois` instead.

Reply:
This is in renderPois()

Reply:
The other one, line 300 ish